Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py

```diff
@@ -11,6 +11,7 @@ def group_by_type(indicators):
     """
 
     grouped = {}
+    indicators = indicators[0]['Contents']
```

Comment:
This direct access to 0 and then to Contents can lead to an exception.
Consider checking the type of object for indicators or try a check indicators[0] before accessing Contents or return a default value from indicators[0].